limitlion/running_counter.py

```diff
@@ -106,20 +106,31 @@ def _get_name(self, name):
                 raise ValueError('Name not specified')
             return name
 
-    def _all_buckets(self, now):
+    def _all_buckets(self, recent_periods=None, now=None):
```

Comment:
I think this function needs a new name now, since it doesn't return all buckets? And maybe we need to designate a term for bucket names (or whatever we want to call them). Maybe `_get_bucket_ids`? And call the bucket names `bucket_ids`? Since we have a function `buckets` that returns `(bucket_id, bucket_count)` pairs, we may consider *that* to be a bucket.

Examples for things in this `_get_bucket_ids`:

`current_bucket` -> `current_bucket_id`
`oldest_bucket` -> `oldest_bucket_id`
`buckets` -> `bucket_ids`

Reply:
Good catch, I will keep it as buckets, but rename `buckets` -> `buckets_counts`, and also the namedtuple `BucketValue` -> `BucketCount`.

I think we could also get rid of the period everywhere, and use `recent_buckets`, and `buckets` in `__init__`